tests/integrationv2/test_happy_path.py

```diff
@@ -116,4 +122,4 @@ def test_s2n_client_happy_path(managed_process, cipher, provider, curve, protoco
     for results in server.get_results():
         assert results.exception is None
         assert results.exit_code == 0
-        assert random_bytes in results.stdout
+        assert random_bytes[1:] in results.stdout
```

Comment:
What happens to the first character?

Reply:
I was seeing problems where debug information was printed out after the first character (from -debug and -tlsdebugext).